INDEX

```diff
@@ -307,5 +307,13 @@
         "category": "collaboration",
         "maintainer": "iXsystems <dev@ixsystems.com>",
         "homepage": "https://iconik.io/"
-    }
+    },
+    "archiwarep5": {
+        "MANIFEST": "archiware.json",
```

Comment:
Please name this correctly to reflect the name of the manifest file which shows up as `archiwarep5.json` as the plugin won't install otherwise.